addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1081,18 +1094,41 @@ QUnit.module('basic_fields', {
             mockRPC: function (route, args) {
                 if (route === "/web/dataset/call_button" && args.method === 'translate_fields') {
                     assert.deepEqual(args.args, ["partner",1,"foo"], 'should call "call_button" route');
-                    return Promise.resolve();
+                    return Promise.resolve({
+                        domain:[],
+                        context:{search_default_name: 'partnes,foo'},
+                    });
+                }
+                if (route === "/web/dataset/call_kw/res.lang/get_installed") {
+                    return Promise.resolve([["en_US"],["fr_BE"]]);
                 }
                 return this._super.apply(this, arguments);
             },
         });
         await testUtils.form.clickEdit(form);
         var $button = form.$('input[type="text"].o_field_char + .o_field_translate');
         assert.strictEqual($button.length, 1, "should have a translate button");
+        assert.strictEqual($button.text(), 'EN', 'the button should have as test the current language');
         await testUtils.dom.click($button);
+        await testUtils.nextTick();
```

Comment:
why?

Reply:
maybe by loading translation_dialog.xml [here](https://github.com/odoo/odoo/blob/12.0/addons/web/static/tests/helpers/test_utils.js#L33), you could get rid of it